### PR TITLE
Misc :: miscellaneous changes for small bugs & test runs

### PR DESCRIPTION
**Pull request by sheyma**
- `test_SurfStatLinMod.py` and `test_SurfStatPeakClus.py` are able to run again.

- bug in `SurfStatEdg.py` is fixed by reverting interp1, ismember, and accum functions in `matlab_functions.py`.

- `need_not_convert` path is added into `tests/test_SurfStatCoord2Ind.py` and `tests/test_SurfStatCoord2Ind.py`, so they are able to run again (removing/keeping these tests are optional).

- earlier work on `SurfStatPCA.py` and `SurfStatInflate.py` are loaded in the `need_not_convert` dir (just to keep them in history).

Screenshot_2020-09-07_15-56-42

### CODE REVIEW

File: surfstat/python/matlab_functions.py [hunk 1]
```diff
@@ -93,4 +72,106 @@ def colon(start,stop,increment=1):
         return r
     elif start == stop or r[-1] + increment == stop:
         r = np.append(r,stop)
-    return r
\ No newline at end of file
+    return r
+
+def accum(accmap, a, func=None, size=None, fill_value=0, dtype=None):
```
Comment (ReinderVosDeWael) [inline surfstat/python/matlab_functions.py:77]: There are bugs in the accum function; I'd remove it entirely and replace all occurrences of it with np.bincount

Comment (sheyma) [reply surfstat/python/matlab_functions.py]: Cool! 
I haven't come across with any bugs in accum. However, trusting on your experience with accum, it is now replaced with np.bincount.

File: surfstat/python/matlab_functions.py [hunk 2]
```diff
@@ -13,38 +14,18 @@ def row_ismember(a, b):
...
-    A : 1D or 2D numpy array 
-    B : 1D or 2D numpy array 
-    rows : logical denoting whether to test for element-wise occurence or row occurence. 
-    
-    Returns
-    -------
-    bool_array : Boolean of the same size as A denoting which elements (or rows) occur in B.
-    locations : Indices of matching elements/rows in A.  
-    """ 
+    # Implementation of MATLAB's ismember() function.
```
[the remark this answers is not in the document]
Comment (sheyma) [reply surfstat/python/matlab_functions.py]: accum is removed, ismember not.
ismember is currently used in `SurfStatPeakClus.py`.
row_ismember was specifically written for` SurfStatResels.py`. 
Unfortunately, replacing ismember with row_ismember does not work in `SurfStatPeakClus.py`. 

yeah, sorry, docstring reverted.

File: surfstat/python/matlab_functions.py [hunk 3]
```diff
@@ -13,38 +14,18 @@ def row_ismember(a, b):
...
-    y : y coordinates of training data.
-    ix : x coordinates of the interpolated points. 
-    kind : type of interpolation; see scipy.interpolate.interp1d for options. 
-    
-    Returns
-    -------
-    iy : interpolated y coordinates.   
-    """ 
-    
+    # Let's call it interp1_mp like matlab-python
```
[the remark this answers is not in the document]
Comment (sheyma) [reply surfstat/python/matlab_functions.py]: this was my mistake.

**Review by ReinderVosDeWael — APPROVED** [review]

**Comment by sheyma** [comment]
Yihuuuuu!!!!!!!!!!!

**Pull request merged**